From 6639ee0a8a3b2f43610288c24605ae6f9e176e62 Mon Sep 17 00:00:00 2001
From: Jungtaek Lim (HeartSaVioR) <kabhwan@gmail.com>
Date: Wed, 12 Dec 2018 19:49:09 +0900
Subject: [PATCH] [BUG-115910][SS] Expose kafka params as field value in KafkaSourceRDD

* This is for enabling custom atlas cluster name to Kafka source/sink in Atlas

Change-Id: I62baf12913e21e318fe64619c1d84e895d46379f
---

diff --git a/external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala b/external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala
index ca05b9e..4be749c 100644
--- a/external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala
+++ b/external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala
@@ -60,7 +60,7 @@
  */
 private[kafka010] class KafkaSourceRDD(
     sc: SparkContext,
-    executorKafkaParams: ju.Map[String, Object],
+    val executorKafkaParams: ju.Map[String, Object],
     offsetRanges: Seq[KafkaSourceRDDOffsetRange],
     pollTimeoutMs: Long,
     failOnDataLoss: Boolean,
